agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -388,6 +388,10 @@ public static class PreviewInstrumentation {
         new DisabledByDefaultInstrumentation();
 
     public DisabledByDefaultInstrumentation vertx = new DisabledByDefaultInstrumentation();
+
+    // this is opt-in because it can cause startup slowness due to expensive matchers
+    public DisabledByDefaultInstrumentation jaxrsAnnotations =
```

Comment:
disabled is default. let's track enabled via Feature statsbeat?

Reply:
added ptal